README.md

```diff
@@ -2,21 +2,16 @@
 
 Signal is a messaging app for simple private communication with friends.
 
-[![Available on the AppStore](http://cl.ly/WouG/Download_on_the_App_Store_Badge_US-UK_135x40.svg)](https://itunes.apple.com/app/id874139669)
+[![Available on the App Store](http://cl.ly/WouG/Download_on_the_App_Store_Badge_US-UK_135x40.svg)](https://itunes.apple.com/us/app/signal-private-messenger/id874139669?mt=8)
```

Comment:
This is a link to the US app store. If we can, we might want to verify that the link works for users who are using app store in other countries.